test/unit/test_adapter.py

```diff
@@ -25,6 +25,7 @@ def setUp(self):
                     'schema': 'dbt_test_schema',
                     'method': 'none',
                     'user': 'trino_user',
+                    'session_properties': {"query_max_run_time": "5d"}
```

Comment:
I am not really sure whether that test is actually relevant.

Reply:
Please do specify multiple session properties in the dictionary.


Can't you check the contents of the session properties dictionary in the test?

Reply:
I meant that if I add random key-value pair test still pass.